examples/tinylora_simpletest.py

```diff
@@ -75,7 +85,7 @@
 lora = TinyLoRa(spi, cs, irq, rst, ttn_config)
 
 while True:
-    data = bytearray(b"\x43\x57\x54\x46")
+    data = bytearray4(b"\x43\x57\x54\x46")
```

Comment:
We'll need to test this out on a device. I'm thinking it might raise an exception for these statements where the bytearray number types like `bytearra4` are used since those are initially defined inside of the try block that will fail on the microcontroller (as intended) due to not having typing.

There are  a few places in the code that could have the same issue if it does work like that. In the cases where they are used as type annotations I think it should be okay because microcontroller will ignore it.

If that is the case we might have to revert them to normal `bytearray` even if mypy strict throws problems about it.

Reply:
Switched all of these back to `bytearray` with an appropriate annotation, e.g.:
```python
data: bytearray4 = bytearray(b"\x43\x57\x54\x46")
```
for this one.